cancer_ai/validator/model_db.py

```diff
@@ -119,7 +119,7 @@ def get_block_timestamp(self, block_number):
             bt.logging.error(f"Error retrieving block timestamp: {e}")
             raise
 
-    def get_latest_models(self, hotkeys: list[str], cutoff_time: float = None) -> list[ChainMinerModel]:
+    def get_latest_models(self, hotkeys: list[str], cutoff_time: float = None) -> list[tuple[ChainMinerModel, str]]:
```

Comment:
I think hotkey should be first, or you could return dict hotkey -> ChainMinerModel, nicer